tests/application/cms/test_models.py

```diff
@@ -632,6 +632,31 @@ def test_file_names(self):
         assert dimension.static_table_file_name == "dimension-guid-table.csv"
 
 
+class TestMeasureModel:
+    def test_versions_to_publish(self):
+
+        measure = MeasureFactory.create()
+        measure_version_1_0 = MeasureVersionFactory.create(
+            measure=measure, version="1.0", status="APPROVED", published_at=datetime(2018, 1, 19).date()
+        )
+        measure_version_1_1 = MeasureVersionFactory.create(
+            measure=measure, version="1.1", status="APPROVED", published_at=datetime(2018, 1, 20).date()
+        )
+        measure_version_1_2 = MeasureVersionFactory.create(
+            measure=measure, version="1.2", status="APPROVED", published_at=datetime(2018, 1, 21).date()
+        )
+        measure_version_2_0 = MeasureVersionFactory.create(
+            measure=measure, version="2.0", status="APPROVED", published_at=datetime(2019, 1, 19).date()
+        )
+        MeasureVersionFactory.create(measure=measure, version="2.1", status="DRAFT", published_at=None)
```

Comment:
If you change all of the `MeasureVersionFactory`'s to use `.build()` rather than create, and then add a single db commit at the end, this test should run a tad faster.

You'll still need to create the measure initially, since that's used for a relationship. But updating the other ones would take us from 6 DB round trips to 2. Every little helps with our tests.